ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -9,7 +9,7 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
+      OS_CLOUD: "openstack"
       OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
```

Comment:
Is this variable used any more? It seems to have been replaced by `OPENSTACK_INSTANCE_FLAVOR`. It is referenced at https://github.com/openshift/release/pull/15992/files#diff-fbda4fe11e77df9ad1e1c33eb9a725bc559ffe175bb43ce91d78ec449ade44ceR13, too.

Reply:
removed. Done